notebooks/parsing_mbox.ipynb

```diff
@@ -40,7 +40,7 @@
   },
```

Comment:
<p>A better name for this function might be "clean_text" or "preprocess_email"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>